controlplane/pkg/apis/connectioncontext/connectioncontext.proto

```diff
@@ -40,5 +57,8 @@ message ConnectionContext {
     repeated Route routes = 5;          /* a list of routes via dst_ip_addr as gateway */
     repeated string excluded_prefixes = 6; /* a list of excluded passed to NSE */
 
-    repeated IpNeighbor ip_neighbors  = 7;
+    repeated IpNeighbor ip_neighbors  = 7; /* A list of ARP table neighbours addresses */
+
+    repeated ExtraPrefixRequest extra_prefix_request = 8;   /* A request for NSE to have provide extra prefixes */
```

Comment:
An extra "have" here. Plese replace with "A request for NSE to provide extra prefixes".